indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskQueue.java

```diff
@@ -1020,24 +1027,45 @@ public Optional<Task> getActiveTask(String id)
   }
 
   /**
-   * List of all active and completed tasks currently being managed by this
-   * TaskQueue.
+   * Polls {@link #activeTasks} for the task with the corresponding {@code taskId}
+   * @param taskId
+   * @return an optional TaskInfo
```

Comment:
```suggestion
   * Gets the {@link TaskInfo} for the given {@code taskId} from {@link #activeTasks} if present,
   * otherwise returns an empty optional.
```